### ISSUE TITLE
TableExpandRow button fails IBM Equal Access A11y Checker

### ISSUE THREAD

**Issue by k8vance88** [carbon-design-system/carbon#6693]
> Browser
Chrome

> Automated testing tool and ruleset
Latest

> Assistive technology used to verify
IBM Equal Access A11y Checker

## Detailed description

Failure is: An input element must have an associated visible label
 
`<button type="button" class="bx--table-expand__button" aria-label="Expand current row">...</button>`

The button is getting flagged by the tool.  

> What version of the Carbon Design System are you using?
10

- Screenshots or code
![image](https://user-images.githubusercontent.com/29546393/90644605-88b72900-e1fa-11ea-85e8-5c682fee93aa.png)

**Comment by tw15egan** [comment]
@dakahn not sure this warning is correct, the warning shows only references to `input` elements. Any action needed on this? 

![Screen Shot 2020-08-19 at 10 14 23 AM](https://user-images.githubusercontent.com/11928039/90668356-d5076680-e204-11ea-8246-982c1896055f.png)

**Comment by dakahn** [comment]
These buttons need visible _readable_ labels to be compliant. This is a valid report @tw15egan

**Comment by joshblack** [comment]
@dakahn the linked rule seems to specifically be for **form controls**, right? [Link](https://www.w3.org/WAI/WCAG21/Techniques/html/H44)

The rule also seems to exclude buttons from this, as well:

![image](https://user-images.githubusercontent.com/3901764/90798680-dce40b00-e2d7-11ea-87b9-3f18bee8a49b.png)

However, we're not in a form and the button does have a label (through `aria-label`). Similarly, doesn't AC itself use this pattern in the screenshot?

**Comment by k8vance88** [comment]
@dakahn is it enough that it has aria-label? https://www.w3.org/TR/WCAG20-TECHS/ARIA14.html

**Comment by tombrunet** [comment]
Conferred with the team, and we agree with Josh that this issue should not apply to buttons. Will track this as a defect in the checker rules: https://github.com/IBMa/equal-access/issues/204

The aria-label was checked as part of a separate rule / success criteria, and that will continue to be checked.

**Comment by tombrunet** [comment]
FYI - you should no longer see this error if you use the `preview` archive of the accessibility checker.

**Comment by tw15egan** [comment]
Thanks for circling back with us @tombrunet and giving us some more background! Going to close this one out as fixed via https://github.com/IBMa/equal-access/pull/205
